package.json

```diff
@@ -18,7 +18,7 @@
     "docker_down": "docker-compose down",
     "docker_build": "docker-compose up -d --build",
     "docker_clean_db": "docker-compose down && docker volume rm educado-backend_mongodb_data && docker-compose up -d --build mongodb",
-    "linting": "eslint --ext .js,.jsx,.ts,.tsx .",
+    "linting": "eslint --ext .js,.jsx,.ts,.tsx . --fix",
```

Comment:
I'm not sure if we should have --fix forced here. Thoughts?

Reply:
yes we should @Jatewo , then it runs fix when you call the linting command

Reply:
I know what it does. But I'm just not sure if we should force it to run with --fix. Now we can't use it to simply check the state of it.